### ISSUE TITLE
Browser won't stay logged in to facebook

### ISSUE THREAD

**Issue by foresto** [LegacyXperia/local_manifests#868]
Ever since installing LegacyXperia, the facebook mobile web site has been frequently requiring me to log in again, despite the fact that I have "Keep site's cookies" checked for both m.facebook.com and www.facebook.com. This is frustrating (and surprising) by itself, but it got worse today: facebook's login page is now telling me that cookies are not enabled on my browser, when they are.

My build: cm-11-20141224-UNOFFICIAL-LegacyXperia-mango

![screenshot_2015-01-14 facebook no cookies](https://cloud.githubusercontent.com/assets/322134/5746496/ecd4d412-9be4-11e4-8b90-462484bd8719.png)

![screenshot_2015-01-14 facebook cookies enabled](https://cloud.githubusercontent.com/assets/322134/5746507/fa37178c-9be4-11e4-9019-ac4bea72445f.png)

![screenshot_2015-01-14 facebook cookies settings](https://cloud.githubusercontent.com/assets/322134/5746511/fdc1abc4-9be4-11e4-8631-18389072fc5a.png)

**Comment by mikeNG** [comment]
Have you enabled 'reduce tracking'? Can't reproduce the issue

**Comment by foresto** [comment]
Yes, "reduce tracking" is set as shown in the screen shot. I'll give it a try with the other two options for that setting.

**Comment by foresto** [comment]
Setting "reduce tracking" to "never delete" seems to avoid the problem. I'll try "delete from all sites that aren't in my whitelist" for a while to see if the bug shows up with that setting.

**Comment by foresto** [comment]
I ran for a week or so with "delete from all sites that aren't in my whitelist", and the browser kept my facebook login. Today, I set it to "also delete all localstorage databases", and then it lost my facebook login. I changed back to "delete from all sites that aren't in my whitelist" and logged in to facebook again, but it lost my login again.

This is pretty strange. I haven't found the pattern yet, but I'll keep trying.

I'm now using cm-11-20150407-UNOFFICIAL-LegacyXperia-mango.
